ui.apps/src/main/javascript/minicart/src/index.js

```diff
@@ -18,15 +18,79 @@ import ApolloClient from 'apollo-boost';
 import { ApolloProvider } from '@apollo/react-hooks';
 
 import Cart from './components/Minicart';
+import { CartProvider } from './utils/state';
+import { useCookieValue } from './utils/hooks';
+import CartInitializer from './utils/cartInitializer';
 
 const App = () => {
     const client = new ApolloClient({
         uri: '/magento/graphql'
     });
 
+    const [, setCartCookie] = useCookieValue('cif.cart');
+
+    const initialState = {
```

Comment:
This should be moved in a separate module since `index.js` is also a root for the account management components.